app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDTOMockkUnitTest.kt

```diff
@@ -64,15 +50,42 @@ class DrinkDTOMockkUnitTest {
         mvm.drinkService = drinkService
     }
 
+    @Test
+    fun searchForCoke_returnsCoke(){
+        whenSearchForCoke()
+        thenResultContainsCoke()
+        thenVerifiedCokeFunctionsEnvoked()
+    }
+
+    @Test
+    fun searchForMariageFreres_returnsMariageFreres(){
+        whenSearchForMariageFreres()
+        thenResultContainsMariageFreres()
+        thenVerifiedMariageFreresFunctionsEnvoked()
+    }
+
+    @Test
+    fun searchForGarbage_ReturnsNothing(){
+        whenISearchForGarbage()
+        thenIGetZeroResults()
+    }
+
     private fun whenSearchForCoke() {
         mvm.fetchDrinks("coke")
     }
 
+    private fun whenSearchForMariageFreres() {
+        mvm.fetchDrinks("Mariage Freres")
+    }
+
+    private fun whenISearchForGarbage() {
+        mvm.fetchDrinks("asdflkj")
+    }
+
     private fun thenResultContainsCoke() {
         var cokeFound = false
 
         mvm.drinks.observeForever {
-            //observe the drink data
```

Comment:
Redundant comment in code. Definitely in the camp for "Clean Code" where functions should clearly be named what they do. In this case, observeForever is pretty self evident on what it does.

Reply:
Yes... one can read this from the code itself.